account/client.go

```diff
@@ -18,76 +21,83 @@ func New(params *stripe.AccountParams) (*stripe.Account, error) {
 	return getC().New(params)
 }
 
+// New creates a new account.
 func (c Client) New(params *stripe.AccountParams) (*stripe.Account, error) {
 	acct := &stripe.Account{}
 	err := c.B.Call(http.MethodPost, "/accounts", c.Key, params, acct)
 	return acct, err
 }
 
-// Get returns the details of an account.
+// Get retrieves the authenticating account.
 func Get() (*stripe.Account, error) {
 	return getC().Get()
 }
 
+// Get retrieves the authenticating account.
```

Comment:
Can anything about the duplicate comments between both calls? Also aren't you missing one `New` since there should be two of those in the file?

Reply:
Unfortunately no ... Golint expects all exported types/functions to be documented. One thing we could do is to possibly just reference the first invocation from the second so that at least we don't have to duplicate the text (might be less change churn when updates are made in the future):

``` go
// New is an alias for Client.New that uses a default client.
```

Maybe that would better?

> Also aren't you missing one New since there should be two of those in the file?

Believe it or not, one of them already had this docstring on it. You can see that the final result looks right if you open the branch's file.

Reply:
gotcha. In that case I think it's better to document than reference the other one because it would display the right string to the user who needs it.